docs/src/test/scala/docs/http/scaladsl/HttpServerWithActorInteraction.scala

```diff
@@ -69,7 +70,7 @@ object HttpServerWithActorInteraction {
             implicit val timeout: Timeout = 5.seconds
 
             // query the actor for the current auction state
-            val bids: Future[Bids] = (auction ? GetBids).mapTo[Bids]
+            val bids: Future[Bids] = (auction ? (replyTo => GetBids(replyTo))).mapTo[Bids]
```

Comment:
Hmm, a bit unfortunate

Reply:
With a typed actor I'd expected being able to do `val bids: Future[Bids] = action ? (GetBids(_))` (or `ask` instead of `?` which makes the param list parenthesis's more obvious), without needing the `mapTo` at all.